### PR TITLE
Sx36 view side bar with multiple options 

### PR DESCRIPTION
**Pull request by AmirMaurice**
@maggie-moheb, @daliamaarek , @Youmna-Salah , @azamzamy , @ahmad3m3 plz review :smile: 
Say Hello to our sideBar :cookie: :sunglasses: 
#48

### CODE REVIEW

File: SmartX/app/src/main/java/com/smartx/cookies/smartx/SideBarAdapter.java [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        else{
+            holder.profile.setImageResource(profile);
+            holder.name.setText(name);
+        }
+    }
+
+    /**
+     * This method returns the number of items present in the list.
+     *
+     * @return
```
Comment (daliamaarek) [inline SmartX/app/src/main/java/com/smartx/cookies/smartx/SideBarAdapter.java]: define what is being returned please

**Comment by daliamaarek** [comment]
Add yourself as an author in all the classes you documented in

**Comment by AmirMaurice** [comment]
@daliamaarek check it again plz :smiley:

**Comment by maggie-moheb** [comment]
test is great :+1:

**Comment by ahmad3m3** [comment]
@AmirMaurice : Could you please add report a problem to the side bar.

**Comment by AmirMaurice** [comment]
@ahmad3m3 check it again

**Comment by ahmad3m3** [comment]
It works perfectly :+1:  @AmirMaurice

**Pull request merged**